CMakeLists.txt

```diff
@@ -233,6 +233,8 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
                     -D_WIN32_WINDOWS=0x502
                     -D_SETUPAPI_VER=0x502
                     -DMINGW_HAS_SECURE_API=1
+                    -DD3D_UMD_INTERFACE_VERSION=0x000C # Vista
+                    -DDXGKDDI_INTERFACE_VERSION=0x1052 # Vista
```

Comment:
Does this need to be set globally? Or can it be limited to win32ss? (unless there are a lot of modules using these headers?)

Reply:
It needs to be set globally because wine uses it, Win32ss uses, soon dxgkrnl will use it

Reply:
OK I guess it's then simpler to copy-paste it in 10+ modules or so.